scripts/delete_totalmobile_jobs.py

```diff
@@ -7,36 +8,28 @@
 from services.totalmobile_service import TotalmobileService
 
 
-def __remove_default_world_id(list_of_active_world_ids: List[str]) -> List[str]:
-    print("Removing the default world id from the retrieved world ids")
-    default_world_id = "c0ffee00-c8d0-499f-8693-8be6ad1dc6ea"
-    return [
-        world_id
-        for world_id in list_of_active_world_ids
-        if world_id != default_world_id
-    ]
-
-
 def __get_active_world_ids(totalmobile_service_local: TotalmobileService) -> List[str]:
-    print("Retrieving world ids")
-    return __remove_default_world_id(
-        totalmobile_service_local.get_world_model().get_available_ids()
-    )
+    print("Retrieving world IDs")
+    return totalmobile_service_local.get_world_model().get_available_ids()
 
 
 def __map_world_id_to_job_reference(
     totalmobile_service_local: TotalmobileService, world_ids: List[str]
 ) -> List[Dict[str, str]]:
     list_of_jobs = []
     for world_id in world_ids:
-        print(f"Retrieving job references for {world_id}")
+        print(f"Retrieving jobs for world ID {world_id}")
         jobs = totalmobile_service_local.get_jobs(world_id)
-        print(f"Retrieved {len(jobs)} for {world_id}")
+        print(f"Retrieved {len(jobs)} jobs for world ID {world_id}")
         for job in jobs:
-            my_dict = {"world_id": world_id, "reference": job["identity"]["reference"]}
-            list_of_jobs.append(my_dict)
+            if job["visitComplete"] is not True:
```

Comment:
You need to add `visitComplete` to https://github.com/ONSdigital/blaise-totalmobile-services/blob/c7fd34dbddb543c8ee293eff7f9a50610f475267/client/optimise.py#L12 to appease the type checker.